src/Parse/ParseUser.php

```diff
@@ -90,7 +90,7 @@ public function isAuthenticated()
   /**
    * Signs up the current user, or throw if invalid.
    * This will create a new ParseUser on the server, and also persist the
-   * session so that you can access the user using ParseUser::getCurrentUser();.
+   * session so that you can access the user using static::getCurrentUser();.
```

Comment:
lets not change this function comment.. it's not valid outside of the class.  Those who are subclassing can use their classname interchangeably... but it also looks like ParseUser::getCurrentUser() would return the subclassed version as well.